app/src/main/java/com/x8bit/bitwarden/ui/platform/base/util/EventsEffect.kt

```diff
@@ -14,9 +17,21 @@ fun <E> EventsEffect(
     viewModel: BaseViewModel<*, E, *>,
     handler: suspend (E) -> Unit,
 ) {
+    val lifecycleOwner = LocalLifecycleOwner.current.lifecycle
```

Comment:
Can we make this a function param, if you add it after `viewModel` but before `handler`, it should have no breaking changes.